app/src/main/java/com/example/varchasdream11/Fragments/DashboardFragment.java

```diff
@@ -61,6 +69,22 @@ public void onCreate(Bundle savedInstanceState) {
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_dashboard, container, false);
+        View root = inflater.inflate(R.layout.fragment_dashboard, container, false);
+
+        FirebaseUser user = FirebaseAuth.getInstance().getCurrentUser();
+        if(user!= null) {
+//            Toast.makeText(this, user+"SIGNED IN", Toast.LENGTH_SHORT).show();
+            String name = user.getDisplayName();
+//            Toast.makeText(this, name+"", Toast.LENGTH_SHORT).show();
+            String email = user.getEmail();
+            Uri photoUrl = user.getPhotoUrl();
+
+            TextView user_name = (TextView) root.findViewById(R.id.etName);
+            user_name.setText(name);
+            TextView mailid = (TextView) root.findViewById(R.id.etEmail);
+            mailid.setText(email);
+            Picasso.get().load(photoUrl).into((ImageView) root.findViewById(R.id.dashboardCardImg));
+        }
```

Comment:
Add an `else statement` and redirect to SignInActivity `if user == null;`

Reply:
Make these positively in next PR.

Reply:
Done